### PR TITLE
relax constraints for aiida-2.0

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

**Comment by ltalirz** [comment]
@kjappelbaum @yakutovicha Could one of you please review this?

By the way, just checking that this repo will continue being maintained, as aiidalab-eln is currently a dependency of aiidalab-widgets-base

File: setup.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     ],
     "description": "Package that integrates AiiDAlab with Electronic Laboratory Notebooks.",
     "install_requires": [
-        "aiida-core>=1.1.0,<2.0.0",
+        "aiida-core>=1.1.0,<3.0.0",
```
Comment (csadorf) [inline setup.json:10]: I believe that this should suffice:
```suggestion
        "aiida-core>=1.1.0,<3",
```

**Review by csadorf — APPROVED** [review]
Please consider to accept my suggestion, but generally approved.

**Comment by csadorf** [comment]
@ltalirz Please rebase this branch and consider to apply my suggestion. Thank you.

**Comment by ltalirz** [comment]
Cheers, done!

**Review by csadorf — APPROVED** [review]

**Comment by csadorf** [comment]
> By the way, just checking that this repo will continue being maintained (with the move of jobs, etc.), as aiidalab-eln is currently a dependency of aiidalab-widgets-base

@kjappelbaum @yakutovicha I would also be interested in the answer to this question. Thanks.

**Comment by kjappelbaum** [comment]
>@kjappelbaum @yakutovicha I would also be interested in the answer to this question. Thanks.

Also happy to do this (sorry, was not available this week). However the caveat for me is, of course, that I'm not involved in AiiDA development and hence less aware of ongoing changes from this side.

**Comment by ltalirz** [comment]
@yakutovicha Could you please drop a new release for this?

Then we can move forward with https://github.com/aiidalab/aiidalab-widgets-base/pull/297

**Comment by yakutovicha** [comment]
@ltalirz I will take care of making a new release today or tomorrow. Need to fix a few things here.

**Pull request merged**